assets/icons/Icon.tsx

```diff
@@ -8,7 +8,7 @@ import React from 'react';
  * used as <Icon type={"close"} />
  */
 
-export type IconType = 'location';
+export type IconType = 'location' | 'green_check' | 'gray_dot';
```

Comment:
are we adding icons in `assets/icons/Icon` or in `app/lib`?

Reply:
should now be in in `app/lib`, this will need to be updated after rebasing!